### DISCUSSION TITLE
Add FreeBSD runner image

### DISCUSSION THREAD

**Discussion by Thefrank** [actions/virtual-environments#7109] [Ideas]
This was requested two years ago: https://github.com/actions/runner-images/issues/2529 and closed due to time/resource constraints 
[to my understanding of: maintenance concerns ]

Since then people have created workarounds: 
https://github.com/actions/runner/issues/385 (this topic predates the issue above)
https://github.com/ChristopherHX/github-act-runner
https://github.com/marketplace/actions/cross-platform-action
https://github.com/myb-project/guide/blob/main/en/gh_runners.md
https://github.com/davidchisnall/act-pot (used here https://github.com/microsoft/snmalloc/pull/360)

These all come at a cost either in significant time to setup or resource cost in using nested VMs

(packer has a FreeBSD, NetBSD, OpenBSD, and even Solaris version: https://developer.hashicorp.com/packer/downloads)

---
 - I have searched for similar discussion topics: https://github.com/actions/runner-images/discussions?discussions_q=freebsd)
 - I can not open this as an issue as those are restricted to a select number of predetermined templates (e,g., request new tool but no request platform)

**Comment by erik-bershel** [comment]
Hey @Thefrank!
We still don't have plans to provide more images mostly due to maintenance concerns.

**Reply by Thefrank** [reply]
Thanks for the quick reply!

I am willing to take the time to, either alone or with the assistance of a code owner here, to make a PR adding image generation for FreeBSD (changes would also like be needed here https://github.com/actions/runner too) . Would such a PR be accepted?

**Reply by mikhailkoliada** [reply]
@Thefrank Hello! We do not accept contributions for OSes other than Ubuntu/Windows/macOS

**Reply by Thefrank** [reply]
@mikhailkoliada thank you for information.
Can this be added somewhere so that people with other OSes don't make my same mistake? (e.g, https://github.com/actions/runner/blob/main/docs/contribute.md + https://github.com/actions/runner-images/blob/main/CONTRIBUTING.md)?

edit: https://github.com/actions/runner/issues/385 should be closed if you get a chance

**Comment by vedranmiletic** [comment]
FWIW, FreeBSD provides [official OCI container images since 14.3-RELEASE](https://www.freebsd.org/releases/14.3R/announce/).
